lib/OpenQA/Worker/Common.pm

```diff
@@ -55,13 +54,13 @@ our $current_host;
 
 my ($sysname, $hostname, $release, $version, $machine) = POSIX::uname();
 
-# global constants
+# local constants
 use constant {
-    STATUS_UPDATES_SLOW => 10,
-    STATUS_UPDATES_FAST => 0.5,
-    MAX_TIMER           => 100,    # It should never be more than OpenQA::WebSockets::Server::_workers_checker threshold
-    MIN_TIMER           => 20,
-    INTERFACE_VERSION => 1,   # Which version the worker follows. Minimal version that will connect to the openqa server
+    MAX_TIMER => 100,    # It should never be more than OpenQA::WebSockets::Server::_workers_checker threshold
+    MIN_TIMER => 20,
+    # Which version the worker follows. Minimal version that will connect to the openqa server.
+    # Needs to match the WEBSOCKET_API_VERSION in the OpenQA::Constants.
+    WEBSOCKET_API_VERSION => 1,
```

Comment:
The idea behind this was to make sure that if the developer changes something on one end (either worker or server), it would to be sure that it would work with the other end. (There's a test that checks if the two values are equal)

Reply:
So the developer will need to update the WEBSOCKET_API_VERSION in two places? sounds like a [code smell](https://blog.codinghorror.com/code-smells/).

Reply:
@foursixnine well this was discussed either in previous PR or via IRC. I can change it back if everyone agrees.

@mudler ^^^